near-contract-standards/src/non_fungible_token/events.rs

```diff
@@ -0,0 +1,149 @@
+use std::fmt;
+
+use near_sdk::serde::{Deserialize, Serialize};
+
+//enum that represents the data type of the EventLogJson.
+//the enum can either be an NftMintLog or an NftTransferLog
```

Comment:
```suggestion
/// Enum that represents the data type of the EventLog.
/// The enum can either be an NftMint or an NftTransfer.
```

When documenting anything exposed, make sure to use `///` so that they show up in rustdocs